tools/Deployment/gulpfile.js

```diff
@@ -218,6 +227,6 @@ gulp.task("dev:release", gulp.series("pack", "publish:azdevops-perf-login", "pub
 
 gulp.task("master:build", gulp.series("clean", "build:release", "e2eTest", "updateGhPage"));
 gulp.task("master:pack", gulp.series("pack"));
-gulp.task("master:release", gulp.series("packAssetZip", "publish:azdevops-prod-login", "publish:azdevops-prod", "publish:gh-release", "publish:gh-asset", "publish:chocolatey"));
+gulp.task("master:release", gulp.series("packAssetZip", "publish:azdevops-prod-login", "publish:azdevops-prod", "publish:gh-release", "publish:gh-asset", "publish:chocolatey", "publish:nuget"));
```

Comment:
Move new task after "publish:azdevops-prod".

Sometimes the "publish:choco" will fail. Move this before can ensure more tasks to succeed in current run.